scripts/emqx_rel_otp_app_overwrite.escript

```diff
@@ -7,19 +7,24 @@
 %%
 %% note, we use NEW to overwrite OLD is because the modified NEW rel file will be overwritten by next 'rebar relup'
 %%
-main([Dir, Profile, RelVsn, BASE_VERSIONS]) ->
-    {ErtsVsn, Overwrites} = get_otp_apps(rel_file(Profile, Dir, RelVsn), RelVsn),
```

Comment:
Could we add `-mode(compile).` to make the error messages more clear?

Reply:
fixed